roles/pm2/manage/tasks/main.yml

```diff
@@ -27,3 +31,5 @@
   tags:
     - restart
     - start
+  when: pm2_config.stat.exists
```

Comment:
This would prevent execution on a first deployment, since the file is not present yet.